package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ueli",
   "productName": "ueli",
-  "version": "8.7.0",
+  "version": "8.8.0",
```

Comment:
Please revert this, because I only do version bumps when publishing a new release.

Reply:
Ok, fixed.